_topic_map.yml

```diff
@@ -93,9 +104,6 @@ Topics:
 - Name: The CI/CD methodology and practice
   File: cicd_gitops
   Distros: openshift-enterprise,openshift-webscale
-- Name: Admission plug-ins
```

Comment:
Is this supposed to be removed?

Reply:
No. Good catch. I wrestled with the topic_map on this PR.